RecoPixelVertexing/PixelTrackFitting/plugins/PixelTrackDumpCUDA.cc

```diff
@@ -71,19 +81,19 @@ void PixelTrackDumpCUDAT<TrackerTraits>::analyze(edm::StreamID streamID,
     cms::cuda::ScopedContextProduce ctx{hTracks};
 
     auto const& tracks = ctx.get(hTracks);
-    auto const* tsoa = tracks.get();
+    auto const* tsoa = &tracks;
     assert(tsoa);
```

Comment:
Now this `assert` and the one below can never fail, so they should be removed, or changed to something that is more useful.